tools/distpartitioning/initialize.py

```diff
@@ -0,0 +1,474 @@
+import os
```

Comment:
the file name is misleading. why is the file called `initialize.py`?

Reply:
Lets call this initialize for now and later change this to an appropriate name, which ever you guys think is appropriate.